config/CLI/trainer.yaml

```diff
@@ -0,0 +1,49 @@
+trainer:
+  fast_dev_run: False
+  # default_root_dir: "/scratch/shared/py4cast/logs/test_cli"
+  num_nodes: 1
+  devices: "auto"
+  max_epochs: 4
+  deterministic: True
+  strategy: "ddp"
+  accumulate_grad_batches: 10
+  accelerator: "auto"
+  logger: 
+    class_path: lightning.pytorch.loggers.TensorBoardLogger
+    init_args:
+      name: "folder_logger"
+      version: "alpha"
+      default_hp_metric: False
```

Comment:
in other projects at AI Lab, we only specify the output directory in the logger part : 
```yaml
trainer:
  logger:
    class_path: lightning.pytorch.loggers.TensorBoardLogger
    init_args:
      default_hp_metric: false
      save_dir: /scratch/shared/bmr/logs/
      name: name_of_experiement
```
Then lightning will automatically create a folder named `/scratch/shared/bmr/logs/name_of_experiement/version_0`. This folder will be linked to the checkpoint callback and contain the checkpoints and the tensorboard artifacts during training. Versions will be automatically incremented when you launch a new run with the same config.
We don't use the default root dir.